website/src/layouts/base/header/MegaMenu.tsx

```diff
@@ -36,13 +36,18 @@ export function MegaMenuListEntry({
     href,
     label,
     className,
+    externalLink = false,
 }: WithClassName<{
     href: string;
     label: string;
+    externalLink?: boolean;
 }>) {
     return (
         <li>
-            <a className={`p-2 hover:underline hover:decoration-4 ${className}`} href={href}>
+            <a
+                className={`p-2 hover:underline hover:decoration-4 ${className} ${externalLink ? 'after:iconify after:mdi--external-link' : ''}`}
```

Comment:
Could you increase the spacing between the icon and the text? It sticks directly to it...
Maybe we need then a div and the anchor for this?

Reply:
Do we want to align the text and the icon vertically?

Reply:
Actually I like that it's quite close to the link. I gave it a 0.5 margin now.

And I centered it.